input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
It's this spectrum, no?
http://iopscience.iop.org/article/10.1086/587736/fulltext/22376.figures.html
For such a spectrum (simple shape, bins not very wide) results don't depend strongly on what you do.
Would suggest to compute `e_ref = sqrt(e_min * e_max)`, i.e. log bin center here and add that as a column, and a one-line comment in the header saying that this is what was done.